package.json

```diff
@@ -15,86 +15,95 @@
+    "@storybook/addon-essentials": "7.4.0",
+    "@storybook/addon-links": "7.4.0",
+    "@storybook/addons": "7.4.0",
+    "@storybook/builder-webpack5": "7.4.0",
+    "@storybook/react": "7.4.0",
+    "@storybook/react-webpack5": "^7.4.0",
+    "@storybook/theming": "7.4.0",
+    "@svgr/cli": "8.1.0",
+    "@types/mapbox-gl": "2.7.13",
+    "@types/node": "20.5.9",
+    "@types/react": "18.2.21",
+    "@types/react-dom": "18.2.7",
+    "@types/react-transition-group": "4.4.6",
+    "@typescript-eslint/eslint-plugin": "6.6.0",
+    "@typescript-eslint/parser": "6.6.0",
+    "babel-loader": "9.1.3",
+    "css-loader": "6.8.1",
+    "dotenv": "16.3.1",
+    "eslint": "8.48.0",
+    "eslint-config-react-app": "7.0.1",
     "eslint-plugin-flowtype": "8.0.3",
-    "eslint-plugin-import": "2.26.0",
-    "eslint-plugin-jsx-a11y": "6.5.1",
-    "eslint-plugin-react": "7.29.4",
-    "eslint-plugin-react-hooks": "4.4.0",
-    "husky": "7.0.4",
-    "lint-staged": "12.3.7",
+    "eslint-plugin-import": "2.28.1",
+    "eslint-plugin-jsx-a11y": "6.7.1",
+    "eslint-plugin-react": "7.33.2",
+    "eslint-plugin-react-hooks": "4.6.0",
+    "husky": "8.0.3",
+    "lint-staged": "14.0.1",
     "ncp": "2.0.0",
-    "prettier": "2.6.2",
-    "react-is": "17.0.2",
-    "rimraf": "3.0.2",
-    "style-loader": "2.0.0",
-    "stylelint": "14.6.1",
-    "stylelint-config-standard": "25.0.0",
+    "postcss": "^8.4.29",
+    "postcss-styled-components": "^0.2.1",
+    "postcss-styled-syntax": "^0.4.0",
+    "prettier": "3.0.3",
+    "react": "^18.2.0",
+    "react-docgen-typescript-plugin": "^1.0.5",
+    "react-dom": "^18.2.0",
+    "react-is": "18.2.0",
+    "rimraf": "5.0.1",
+    "storybook": "^7.4.0",
+    "style-loader": "3.3.3",
+    "stylelint": "15.10.3",
+    "stylelint-config-standard": "34.0.0",
     "stylelint-config-styled-components": "0.1.1",
-    "stylelint-processor-styled-components": "1.10.0",
-    "typescript": "4.6.3",
-    "webpack": "4.46.0",
-    "webpack-cli": "4.9.2"
+    "stylis": "^4.3.0",
+    "typescript": "5.2.2",
+    "webpack": "5.88.2",
+    "webpack-cli": "5.1.4"
   },
   "optionalDependencies": {
-    "date-fns": "2.28.0",
-    "mapbox-gl": "1.13.2",
+    "date-fns": "2.30.0",
+    "mapbox-gl": "2.15.0",
     "normalize.css": "8.0.1",
     "polished": "4.2.2",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-mapbox-gl": "5.1.1",
-    "react-transition-group": "4.4.2",
-    "styled-components": "5.3.5"
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-map-gl": "7.1.5",
+    "react-transition-group": "4.4.5",
+    "styled-components": "6.0.7"
   },
   "peerDependencies": {
     "date-fns": ">=2.28",
-    "mapbox-gl": ">=1.13",
+    "mapbox-gl": ">=2.15",
     "normalize.css": ">=8.0",
     "polished": ">=4.2",
     "react": ">=17.0",
     "react-dom": ">=17.0",
-    "react-mapbox-gl": ">=5.1",
+    "react-map-gl": ">=7.1",
     "react-transition-group": ">=4.4",
     "styled-components": ">=5.3"
```

Comment:
Should we bump this peer dependency to >=6.0.0, like with the optional dependency above?

Reply:
"This" dependency being `styled-components`

Reply:
yes